command.go

```diff
@@ -31,6 +34,7 @@ type Command struct {
 func init() {
 	flag.StringVar(&Runtime, "runtime", "", "Path of the desired Kata Runtime")
 	flag.IntVar(&Timeout, "timeout", 5, "Time limit in seconds for each test")
+	flag.StringVar(&Hypervisor, "hypervisor", "", "The hypervisor currently being used with Kata")
```

Comment:
this is ok now, but we'll need to update #2396

Reply:
If you want to merge #2396 first. I can rebase and adjust the PR accordingly. Just let me know. I am fine either way.

Reply:
I think we can first merge this one, we are still figuring out some other issues with go 1.13.8, so I think that one will be there a little longer. 
wdyt @devimc ?

Reply:
sounds good, let's merge this and rebase #2396

Reply:
@jodh-intel Here is the discussion on how to handle the conflict with #2396. Please share if you have additional thoughts. Thank you.